cmd/podman/varlink/io.podman.varlink

```diff
@@ -1120,6 +1138,9 @@ method GetPodsByContext(all: bool, latest: bool, args: []string) -> (pods: []str
 # LoadImage allows you to load an image into local storage from a tarball.
 method LoadImage(name: string, inputFile: string, quiet: bool, deleteFile: bool) -> (reply: MoreResponse)
 
+# GetEvents returns known libpod events filtered by the options provided.
+method GetEvents(options: EventInput) -> (events: Event)
```

Comment:
I assume this only makes sense when it supports `more`? Please note that in the comment.

Reply:
indeed, fixed.

Reply:
Also, please put the fields of the `EventInput` type into this call. It's awkward to call otherwise.